x-pack/platform/plugins/shared/cases/server/saved_object_types/cases/model_versions.ts

```diff
@@ -86,3 +86,31 @@ export const modelVersion2: SavedObjectsModelVersion = {
     forwardCompatibility: casesSchemaV2.extends({}, { unknowns: 'ignore' }),
   },
 };
+
+/**
+ * Adds timing metrics to the cases SO.
+ */
+export const modelVersion3: SavedObjectsModelVersion = {
+  changes: [
+    {
+      type: 'mappings_addition',
+      addedMappings: {
+        in_progress_at: {
+          type: 'date',
+        },
+        time_to_acknowledge: {
+          type: 'unsigned_long',
+        },
+        time_to_investigate: {
+          type: 'unsigned_long',
+        },
+        time_to_resolve: {
+          type: 'unsigned_long',
+        },
```

Comment:
Is the intention to search over these at some point? I may have missed it in the code, but I'd recommend adding mappings as lazily as possible.

Some context from dev docs [docs](https://docs.elastic.dev/kibana-dev-docs/tutorials/saved-objects#mappings):

```
Do not use field mappings like you would use data types for the columns of a SQL database.... Only specify field mappings for the fields you wish to search on or query.

By specifying dynamic: false in any level of your mappings, Elasticsearch will accept and store any other fields even if they are not specified in your mappings.
```

Reply:
At the moment, there is no use case for searching/filtering those fields, so I would say to remove them from the mappings.

Reply:
Just to double-check. Not having these mappings here will still allow clients to create dashboards with case metrics?

Reply:
Yes, these fields are needed, so they will be copied to the analytics indices.